pywinauto/controls/atspiwrapper.py

```diff
@@ -40,7 +40,7 @@
 from ..base_wrapper import BaseWrapper
 from ..base_wrapper import BaseMeta
 
-from ..atspi_element_info import AtspiElementInfo
+from pywinauto.linux.atspi_element_info import AtspiElementInfo
```

Comment:
Relative imports are more preferable. By the way, `atspiwrapper.py` could be moved to `linux` folder.